Lesson11/merve/index.html

```diff
@@ -0,0 +1,46 @@
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <nav id="navbar" class="nav">
+  <a href class="a--nav-selector" id="profile-link" target="_blank"> ABOUT</a>
+  <a href="project-section" class="a--nav-selector">WORK</a>
+  <a href="contact-section" class="a--nav-selector">CONTACT</a>
+  </nav>
+  <body>
+  <main id="welcome-section" class="welcome-section">
+  <h1 class="welcome-section-header">Hello I am Merve</h1>
+  <p class="welcome-section-header-info">a frontend developer</p>
+  </main>
+  <div id="project-section" class="project-section">
+  <h2 class="project-section-header">There are some of my projects</h2>
+  <div class="projects-section-content">
+  <a href="" class="btn project"><img class="project-image"/></a>
+  <a href="" class="btn project"><img class="project-image"/></a>
+  <a href="" class="btn project"><img class="project-image"/></a>
+  <a href="" class="btn project"><img class="project-image"/></a>
+  <a href="" class="btn project"><img class="project-image"/></a>
+  <a href="" class="btn project"><img class="project-image"/></a>
+  </div></div>
+  <div class="contact-section">
+    <h2 class="contact-section-header">Let's work together</h2>
+    <div class="contact-section-content">
+      <a href="https://www.facebook.com/freecodecamp" target="_blank" class="contact-details">
+        <i class="fa fa-facebook-square"></i> Facebook
+      </a>
+      <a id="profile-link" href="https://github.com/freecodecamp" target="_blank" class="contact-details"><i class="fa fa-github"></i> GitHub</a>
+      <a href="https://twitter.com/freecodecamp" target="_blank" class="contact-details"><i class="fa fa-twitter"></i> Twitter</a>
+      <a href="mailto:example@example.com" class="contact-details"><i class="fa fa-at"></i> Send a mail</a>
+      <a href="tel:555-555-5555" class="contact-details"><i class="fa fa-mobile"></i> Call me</a>
+    </div>
+  </div>
+  <footer>
+  <p> This is a personal Portfolio website.</p>
+  <p> All rights reserved.</p>
+  </footer>
+  </body>
+  
```

Comment:
Closing HTML tag missing
```suggestion
  </html>
```